Engine/PSScriptAnalyzer.psm1

```diff
@@ -9,15 +9,13 @@ $PSModuleRoot = $PSModule.ModuleBase
 
 # Import the appropriate nested binary module based on the current PowerShell version
 $binaryModuleRoot = $PSModuleRoot
-if ($PSVersionTable.PSVersion.Major -eq 7 ) {
-    $binaryModuleRoot = Join-Path -Path $PSModuleRoot -ChildPath "PSv$($PSVersionTable.PSVersion.Major)"
-}
-elseif ($PSVersionTable.PSVersion.Major -eq 6 ) {
+[Version] $minimumPowerShellCoreVersion = '7.0.3'
```

Comment:
I chose to bump not just from `6.2.4` to `7.0.0` but also to `7.0.3` in anticipation of PR #1597 getting merged, which will bump the minimum version to `7.0.3`. Since `7.0.2` is not supported anyway because `7.0.3` is already more than a month old, this should not be an issue.